.github/release_plan.md

```diff
@@ -24,10 +24,8 @@
 -   [ ] Update the [`release` branch](https://github.com/microsoft/vscode-jupyter/branches)
     -   [ ] If there are `release` branches that are two versions old (e.g. release-2020.[current month - 2]) you can delete them at this time
 -   [ ] Update `main` after creating the release branch. (Warning: this should happen right after creating the release branch. If this is deferred till later, the `main` and `release` branches can diverge significantly, which may cause merge conflicts.)
-    -   [ ] Bump the version number to the next monthly ("YYYY.MM.0") release in the `main` branch
-        -   [ ] `package.json`
-        -   [ ] `package-lock.json`
     -   [ ] Merge the changes from release (Changelog, delete news, ThirdPartyNotices) into `main` branch
+    -   [ ] [Turn off automatic uploads for insider builds from main](https://github.com/microsoft/vscode-jupyter/blob/f05fedf399d34684b408245ba27bc29aa25c13f6/.github/workflows/build-test.yml#L73). This prevents stable customers from getting insiders builds.
```

Comment:
We agreed on this, but I still think we can leave the automatic uploads for insiders on. They would still be using the old version and engine numbers.